README.md

```diff
@@ -48,3 +52,16 @@ _These are single words with short sentence fragment descriptions. (5 max)_
 * **OPENNESS** &ndash; Make decisions and coordinate openly and transparently  
 
 _[also, note useful **elevator pitches** -- the “all-in-one” sentences that efficiently go from the problem to the solution]_
+
+## Projects
+
+We coordinate a number of projects and sub-components in the [Data Together github org](https://github.com/datatogether). This is a short list of those projects. For a complete list of all repositories, visit https://github.com/datatogether **Please join in!**
+
+* [Learning](https://github.com/datatogether/learning) - A repository where we are gathering learning materials about Data Together and the surrounding topics.
+* [Platform](https://archivers.co):  The beta platform at https://archivers.co is one example of where we're going. The code for that platform lives at https://github.com/datatogether/webapp
```

Comment:
I'm in full support of positioning all of our technical work as a reference implementation

Reply:
I'm fine with that as well, but as I read the vision our scope is radically altered, and I'm concerned about our ability to create the social/tech infra to support the unfolding of the new scope _and_ extend the reference implementation to our own case (EDGI's case I guess)

Reply:
Let's find a way to balance the vision statement so it's clear that the core of what we're committing to actually do is to build the best possible tools for orgs like EDGI to satisfy their needs. The tool-building is inspired and informed by a broader vision, but we have to make software that gracefully solves real-world problems through interactions that people (and the orgs around them) will love.